### PR TITLE
Fixed DnD Add-on to use resourceID #807

### LINKED ISSUE #807: Can't use the next resourceId when item is dropped
#### Do you want to request a _feature_ or report a _bug_?
_bug_

#### What's the current behavior?
It seems that a next resource id is not used in the Drag and Drop addon.
https://github.com/ehahn9/react-big-calendar/blob/f730f2fee7cf4b99274542df37e0b9a446c17e8e/src/addons/dragAndDrop/DropWrappers.js#L151

#### What's the expected behavior?
I expect it should be use next resource ID like this commits demo.
https://github.com/intljusticemission/react-big-calendar/commit/0c0b17c045a65abaa84f14e91254e04cf34a1873#diff-4f25ee19436cd5ed38d0bce39dbd7f8aR114

### PR DESCRIPTION
**Pull request by mura-**
I tried to fix #807 .
Please check this PR 🙏

### CODE REVIEW

**Comment by SLYtiger16** [comment]
I have installed and tested this update ... aside from a pre-existing issue about dropping events [#832](https://github.com/intljusticemission/react-big-calendar/issues/832) this works well amd as expected ... thank you!

**Comment by mura-** [comment]
@SLYtiger16 Thanks for your testing!

File: src/addons/dragAndDrop/DropWrappers.js [hunk 1]
```diff
@@ -151,6 +153,7 @@ function createDropWrapper(type) {
           onEventDrop({
             event,
             ...getEventDropProps(start, end, value, droppedInAllDay),
+            [resourceAccessor]: resource,
```
Comment (jquense) [inline src/addons/dragAndDrop/DropWrappers.js]: `resourceAccessor` may be function, i would just call this `resource` since it's not hte event object.

Comment (mura-) [reply src/addons/dragAndDrop/DropWrappers.js]: @jquense Thanks!
Every accessors seem to be String.
https://github.com/intljusticemission/react-big-calendar/pull/833/files#diff-6cf28b0e3b0b32fa18e19efed78ee8b8R80


And... I expect `start` and `end` should support Accessor however, I didn't fix this PR. I think we have to fix on another issue.

Comment (jquense) [reply src/addons/dragAndDrop/DropWrappers.js]: the defaults are strings but the prop type allows functions. The other ones are already correct, and don't assume it's a string. It's just your usage here that's not right.

Comment (mura-) [reply src/addons/dragAndDrop/DropWrappers.js]: @jquense Sorry for replying so late. 
I'll check and solve that on this week.

Comment (mura-) [reply src/addons/dragAndDrop/DropWrappers.js]: @jquense 

I have a question.
Do I have to support resourceAccessor on DnD?

`start` and `end` seem to be fixed.
https://github.com/mura-/react-big-calendar/blob/614f01704c828f0f500ba166d13308daff75650c/src/addons/dragAndDrop/DropWrappers.js#L32

If I don't have to support resourceAccessor, What should I use the resource key name?

Comment (jquense) [reply src/addons/dragAndDrop/DropWrappers.js]: you only need it if you actually need to get the value from the user's event object. the returned value here is **not** an event object iso you don't need it here. I'd just call this `resourceId` if that's what it is

Comment (mura-) [reply src/addons/dragAndDrop/DropWrappers.js]: @jquense Thanks a lot. I will fix it.

**Comment by mura-** [comment]
@jquense 
I have fixed resource's key. please take a look 🙏

**Pull request merged**